packages/sync/src/syncInterface.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  AttachmentID,
+  AttachmentMIMEType,
+  Event,
+  EventID,
+} from "@withorbit/core2";
+
+export interface SyncInterface {
+  // This ID is used to track state associated with the remote destination associated with this interface. Different destinations (e.g. test servers, production server, local host) should have different IDs.
+  id: string;
```

Comment:
Is this used for logging purposes or do you expect some utility from this metadata in the future?

Reply:
We use it when storing the last event ID sent/received to/from the sync destination. An ID is necessary to keep different state for different sync destinations (e.g. perhaps you're syncing to both a local server and the production server).

See the next PR for how it's used:
https://github.com/andymatuschak/orbit/pull/232/files#diff-5b6b6441fbdb9546805cc316cbf46bd767f35dd9e60db2897beab1d462b561c3R17

I improved the comment here.